There is no commited version of data/time formats for as locale,
but the uncommited patters looks are sufficient

--- cldr_core.orig/common/main/as.xml
+++ cldr_core/common/main/as.xml
@@ -75,22 +75,22 @@
 				<dateFormats>
 					<dateFormatLength type="full">
 						<dateFormat>
-							<pattern draft="unconfirmed">EEEE, d MMMM, y G</pattern>
+							<pattern>EEEE, d MMMM, y G</pattern>
 						</dateFormat>
 					</dateFormatLength>
 					<dateFormatLength type="long">
 						<dateFormat>
-							<pattern draft="unconfirmed">d MMMM, y G</pattern>
+							<pattern>d MMMM, y G</pattern>
 						</dateFormat>
 					</dateFormatLength>
 					<dateFormatLength type="medium">
 						<dateFormat>
-							<pattern draft="unconfirmed">dd-MM-y G</pattern>
+							<pattern>dd-MM-y G</pattern>
 						</dateFormat>
 					</dateFormatLength>
 					<dateFormatLength type="short">
 						<dateFormat>
-							<pattern draft="unconfirmed">d-M-y GGGGG</pattern>
+							<pattern>d-M-y GGGGG</pattern>
 						</dateFormat>
 					</dateFormatLength>
 				</dateFormats>
